### PR TITLE
Add camel-jetty wrapper

### PR DESCRIPTION
**Pull request by f2par0**
camel-jetty wrapper

### CODE REVIEW

**Review by essobedo — COMMENTED** [review]
I don't believe that forcing the version of Jetty to CXF is the right approach in this case, we need 2 features one for Jetty 11 and another for Jetty 12

**Comment by f2par0** [comment]
kept  2 jetty features, one for version 11 and one for version 12

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -97,6 +97,21 @@
         <bundle dependency="true">mvn:jakarta.servlet/jakarta.servlet-api/5.0.0</bundle>
     </feature>
 
+    <feature name="jetty12" version="${jetty-version}">
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: Let's keep `jetty` as name as we did for the 2 versions of the feature of `http-client`

Comment (f2par0) [reply features/src/main/feature/camel-features.xml]: done, I had tried by fixing the version, it did not work. With version range, it's working

**Review by essobedo — APPROVED** [review]
Nice one 👍 thx

**Pull request merged**